Tasks/Maven/moduleanalysis.ts

```diff
@@ -0,0 +1,7 @@
+import ar = require('./analysisresult');
+// Data class for supporting Maven projects with more than one module
+export class ModuleAnalysis {
+    moduleName: string;
+    rootFolder: string;
+    analysisResults:any = {}; // To be used as a dictionary
+}
```

Comment:
New data class representing a Maven module. It has fields for the module name, the location of the root folder, and a dictionary of {string, AnalysisResult} used to associated toolName(s) with their analysisResult(s).